CRM/Utils/Check/Component/Env.php

```diff
@@ -935,4 +935,23 @@ public function checkMysqlUtf8mb4() {
     return $messages;
   }
 
+  public function checkMysqlVersion() {
+    $messages = [];
+    $version = CRM_Utils_SQL::getDatabaseVersion();
+    $minRecommendedVersion = CRM_Upgrade_Incremental_General::MIN_RECOMMENDED_MYSQL_VER;
```

Comment:
This needs to be min version to address the comments Tim made - with that changed this is mergeable but I feel follow ups should happen to get messaging to move people off 5.5 & 5.6 - probably starting with a gitlab & dev-list email